test/skate/detours/trip_modification_publisher_test.exs

```diff
@@ -29,15 +30,15 @@ defmodule Skate.Detours.TripModificationPublisherTest do
 
   @tag "Test.Integration": :mqtt
   test "can publish as draft via metadata" do
-    {:ok, reader_pid} = MqttConnection.start_link(["trip_modifications/+/trip_modification"])
+    {:ok, reader_pid} = MqttConnection.start_link(["trip_modifications/+/+"])
```

Comment:
**discussion**/suggestion(non-blocking&low-priority): this is good, but do you think it'd be useful to be explicit about our subscriptions?

I'm on the fence about this but thinking of [MQTT [...]Wildcard subscriptions best practices](https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices/), 
> ### Avoid Subscribing to Wildcards (`#`)
> Sometimes, it is necessary to subscribe to all messages that are transferred over the broker. For example, to persist all messages into a database. Do not subscribe to all messages on a broker by using an MQTT client and subscribing to a multi-level wildcard. Frequently, the subscribing client is not able to process the load of messages that results from this method (especially if you have a massive throughput)
>
> –– https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices/

The heading has the `#` wildcard and not the `+` wildcard, and this seems to be talking about multi level wildcards, but we now have two wildcards, and I _wonder_ if we should consider being more explicit both for readability and as a ~"contract"?

Anyway, none of this is important enough to block over merging.

---

```suggestion
    {:ok, reader_pid} =
      MqttConnection.start_link([
        "trip_modifications/+/trip_modification",
        "trip_modifications/+/shape"
      ])
```

Reply:
[Makes sense! I didn't notice that it was an array](https://github.com/mbta/skate/pull/2688/commits/88d2873ea5905d595fa486291fcfd3b4a597f8de)